spec.md

```diff
@@ -953,6 +953,20 @@ existence of property types does not affect hardware functionality or the
 hardware ABI. For example, it is valid to drop property types from the IR
 completely.
 
+Property types are legal in the following constructs:
+
+* Port declarations on modules and external modules
+
+### Integer Property Types
+
+Integer properties represent arbitrary bit width signed integer values with a
+two's complement representation.
```

Comment:
Why does the two's complement encoding matter?

Reply:
I guess it doesn't matter, but it seemed worth specifying. I guess if we ever had arithmetic or boolean operations (which I'm not planning anytime soon), it would matter. I'm happy to leave it off. I looked at the Python docs about the builtin int, and it just mentions arbitrary precision at first (is precision a better term? we use bit width elsewhere in this doc). Later, the Python docs mention two's complement, but only when discussing boolean operations on ints.

Reply:
It would matter if bitwise boolean operations are defined on integer properties. I'm not sure we need that, yet. For arithmetic, I'm not sure it matters as long as integers behave like integers.

Interesting find about Python integers! It looks like that is pedantically talking about the behavior of how integers are interpreted by bitwise operators, not the underlying storage:

> Bitwise operations only make sense for integers. The result of bitwise operations is calculated as though carried out in two’s complement with an infinite number of sign bits.

My nit here is that I don't think it makes sense to be prescriptive about the _storage_ of integers in the specification when that storage is likely invisible to a user.